js/jquery/form-steps.js

```diff
@@ -0,0 +1,85 @@
+import $ from 'jquery'
```

Comment:
What is the prupose of including this JS component within the toolkit? I think it can not be used in a real app since that would require a lot more logic, wouldn't it?

Reply:
I don't get the question. I added a JS module as any other javascript module. The goal is to allow sliding the steps on smartphone.
I tried a CSS only implementation but it makes the markup a lot more complicated for a seldom used feature.